code/game/objects/items/weapons/alien_specific.dm

```diff
@@ -50,30 +50,35 @@
 	reagents.check_and_add("water", volume, 2 * coeff)
 
 /obj/item/reagent_containers/spray/alien/acid
-	name = "acid synthesizer"
-	desc = "squirts burny liquids."
+	name = "polyacid synthesizer"
+	desc = "Sprays concentrated polyacid."
 	icon = 'icons/mob/alien.dmi'
 	icon_state = "borg-spray-acid"
 	list_reagents = list("facid" = 125, "sacid" = 125)
 
 /obj/item/reagent_containers/spray/alien/acid/cyborg_recharge(coeff, emagged)
 	reagents.check_and_add("facid", volume / 2, 2 * coeff) // Volume / 2 here becuase there should be an even amount of both chems.
 	reagents.check_and_add("sacid", volume / 2, 2 * coeff)
-
-/obj/item/reagent_containers/spray/alien/stun
+/*
+/obj/item/reagent_containers/spray/alien/stun	// This doesn't actually function due to ether not having a touch reaction. I have an idea for what to replace this with but it will take some doing.
```

Comment:
```suggestion
/obj/item/reagent_containers/spray/alien/stun	// TODO: This doesn't actually function due to ether not having a touch reaction.
```
Dont leave poor souls from the future wondering what was your plan without saying anything about it if this stays :(